app/helpers/route.py

```diff
@@ -0,0 +1,21 @@
+from functools import wraps
+
+
+def prefix_route(route_decorator, prefix='', fmt='{prefix}{route}'):
+    '''Defines a new route decorator with a prefix.
+
+    Args:
+        route_decorator: route decorator to enhanced
+        prefix: prefix to add to the route
+        fmt: string format to use for adding the prefix
+
+    Returns:
+        new decorator with prefixed route.
+    '''
+
+    @wraps(route_decorator)
+    def newroute(route, *args, **kwargs):
+        '''New function to prefix the route'''
+        return route_decorator(fmt.format(prefix=prefix, route=route), *args, **kwargs)
+
+    return newroute
```

Comment:
We are currently trying another approach to set the prefix in infra, so for the moment I would not add this to the template.